src/Draggable/Plugins/Announcement/Announcement.js

```diff
@@ -0,0 +1,162 @@
+const onInitialize = Symbol('onInitialize');
+const onDestroy = Symbol('onDestroy');
+const announceEvent = Symbol('announceEvent');
+const announceMessage = Symbol('announceMessage');
+
+const ARIA_RELEVANT = 'aria-relevant';
+const ARIA_ATOMIC = 'aria-atomic';
+const ARIA_LIVE = 'aria-live';
+const ROLE = 'role';
+
+export const defaultOptions = {
+  expire: 7000,
+  'drag:start': 'Picked up draggable element',
+  'drag:stop': 'Dropped draggable element',
+};
+
+/**
+ * Announcement plugin
+ * @class Announcement
+ * @module Announcement
+ */
+export default class Announcement {
+
+  /**
+   * Announcement constructor.
+   * @constructs Announcement
+   * @param {Draggable} draggable - Draggable instance
+   */
+  constructor(draggable) {
+
+    /**
+     * Draggable instance
+     * @property draggable
+     * @type {Draggable}
+     */
+    this.draggable = draggable;
+
+    /**
+     * Plugin options
+     * @property options
+     * @type {Object}
+     */
+    this.options = {
+      ...defaultOptions,
+      ...this.getOptions(),
+    };
+
+    /**
+     * Original draggable trigger method. Hack until we have onAll or on('all')
+     * @property originalTriggerMethod
+     * @type {Function}
+     */
+    this.originalTriggerMethod = this.draggable.trigger;
+
+    /**
+     * Live region element
+     * @property liveRegion
+     * @type {HTMLElement}
+     */
+    this.liveRegion = createRegion();
+
+    this[onInitialize] = this[onInitialize].bind(this);
+    this[onDestroy] = this[onDestroy].bind(this);
+  }
+
+  /**
+   * Attaches listeners to draggable
+   */
+  attach() {
+    this.draggable
+      .on('draggable:initialize', this[onInitialize]);
+  }
+
+  /**
+   * Detaches listeners from draggable
+   */
+  detach() {
+    this.draggable
+      .off('draggable:destroy', this[onDestroy]);
+  }
+
+  /**
+   * Returns passed in options
+   */
+  getOptions() {
+    return this.draggable.options.announcements || {};
+  }
+
+  /**
+   * Announces event
+   * @private
+   * @param {AbstractEvent} event
+   */
+  [announceEvent](event) {
+    const message = this.options[event.type];
+
+    if (message && (typeof message === 'string')) {
+      this[announceMessage](message);
+    }
+
+    if (message && (typeof message === 'function')) {
+      this[announceMessage](message(event));
+    }
+  }
+
+  /**
+   * Announces message to screen reader
+   * @private
+   * @param {String} message
+   */
+  [announceMessage](message) {
+    const element = document.createElement('div');
+    element.innerHTML = message;
+    this.liveRegion.appendChild(element);
+    return setTimeout(() => {
+      this.liveRegion.removeChild(element);
+    }, this.options.expire);
+  }
+
+  /**
+   * Initialize hander
+   * @private
+   */
+  [onInitialize]() {
+    this.draggable.trigger = (event) => {
+      this[announceEvent](event);
+      this.originalTriggerMethod.call(this.draggable, event);
+    };
+
+    document.body.appendChild(this.liveRegion);
```

Comment:
The issue with dynamically adding the live region is that some AT do not recognize the region _unless_ it's already present in the DOM on page load. I can run some tests to see if this has been changed, but I'd recommend calling this line right away, perhaps in `constructor()` and leaving it in the DOM.

Reply:
Oh, I didn't realize! It's been working on Chrome/Safari for the mac so far, but I wouldn't be surprised if there are browser incompatibilities.... 😆 

> but I'd recommend calling this line right away

At some point I was thinking of making only one live region, which is then shared by all draggable instances on the page. Could look something like this:

```js
// store as local variable, instead of instance variable
const liveRegion = createLiveRegionElement();

// append at earliest point
document.addEventListener('DOMContentLoaded', () => document.body.appendChild(liveRegion));

// Announcement class would then use the local variable, so it's shared between instances
class Announcement {}
```

And the code would live inside this file, but outside of the class declaration. There would be no way to add the live region earlier than that

Reply:
> At some point I was thinking of making only one live region, which is then shared by all draggable instances on the page.

I like it. And yeah, the earlier it can be added the better.